offer.yaml

```diff
@@ -0,0 +1,24 @@
+applications:
```

Comment:
Why this offer file?

Reply:
deleted, it was for cos-lite deployment without changing executing cd

Reply:
I still see it here